tests/js-tests/main.js

```diff
@@ -117,7 +117,15 @@ cc.game.onStart = function(){
     }
     else
     {
-        cc.loader.resPath = '../cpp-tests/Resources'
+        // js-test use cpptest resource in debug mode , and in the release mode, console will copy the resource into the res dir
+        // so the respath will modify to res,
+        if (cc.game.config[cc.game.CONFIG_KEY.engineDir] !== "frameworks/cocos2d-html5") {
```

Comment:
You should use `cc.game.config[cc.game.CONFIG_KEY.engineDir] === undefined` or `=== ""` if it's initialized
I don't know which case it's in, but don't hard code the url, it may be changed.